### DISCUSSION TITLE
Where is the _green_ box-arrow?

### DISCUSSION THREAD

**Discussion by velle** [bernhard-42/vscode-ocp-cad-viewer#108] [Q&A] [answered]
In https://github.com/bernhard-42/vscode-ocp-cad-viewer?tab=readme-ov-file#running-code-using-jupyter, it says:

> Start the OCP CAD Viewer by pressing the green box-arrow button in the "Viewer Manager" section of the OCP CAD Viewer sidebar

I spent more than a few minutes looking for this button :)

Do you mean this one?

![image](https://github.com/user-attachments/assets/ac0b7ffb-167e-4922-8f4d-79ca1abbe7cd)

It's not green. Was it green in other versions? Or is it green if something is installed differently than on my setup?

Anyway, the button I was looking for is only visible when the `ocp_vscode` line is selected, or if the mouse is hovering over. 

![image](https://github.com/user-attachments/assets/25c11e99-677d-4e94-87e3-f8791a4676c8)

**Comment by bernhard-42** [comment, marked as answer]
Ha, it was green before I had to switch from png to svg icons (VS Code forgot in one release to render png icons). However, VS Code removes colors from svg, which it didn't do for png icons.
I actually missed that it was mentioned in the docs as "green".
So all seems to be correct on your setup. Unfortunately, VS Code in the meantime only shows these icons when you hover or the line is selected.
Whenever you program for a platform like VS Code, you'll again and again get surprised with its changes :-)

**Comment by bernhard-42** [comment]
I changed the Readme accordingly, removed the "green" at 4 places and added each time in brackets that you need to hover over the entry.
Hope that is clear now.
